.gitignore

```diff
@@ -0,0 +1,3 @@
+*venv
+*.idea
+*.DS_Store
```

Comment:
Adding a .gitignore just cleans up the repo a bit. If you want to go this route you'll also have to un-track the files you've already commited to Github:

`git rm -r --cached {filename}`.